content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+elements. The elements are organized into four categories: :ref:`studio/pdf-reports/elements/block`,
+:ref:`studio/pdf-reports/elements/inline`, :ref:`studio/pdf-reports/elements/table`, and
+:ref:`studio/pdf-reports/elements/column`.
+
+.. _studio/pdf-reports/elements/block:
+
+Block
+-----
+
+Block elements start on a new line and occupy the full width of the page.
+
+.. tip::
+   You can set an element's width by selecting it and going to the :guilabel:`Options` tab if you do
+   not want it to use the full width.
+
+- :guilabel:`Text`: add any text using small font size by default.
+
+- :guilabel:`Title Block`: add any text using larger font size by default.
+
+- :guilabel:`Image`: add an image. You can either upload one from your device, add one from
+  an URL, or select one already existing on your database.
+
+- :guilabel:`Field`: dynamically add a field's value.
+
+- :guilabel:`Field & Label`: to dynamically add a field's value and label.
+
+- :guilabel:`Address Block`: to dynamically add the values, if any, of a contact's: *Name*,
+  *Address*, *Phone*, *Mobile*, and *Email*.
+
+  .. image:: pdf_reports/address-block.png
+     :align: center
+     :alt: Example of an Address Block
+
+.. _studio/pdf-reports/elements/inline:
+
+Inline
+------
+
+Inline elements are used around other elements. They do not start on a new line and only occupy the
+required width.
+
+.. tip::
+   You can set an element's width and margins by selecting it and going to the :guilabel:`Options`
+   tab.
+
+- :guilabel:`Text`: add any text using small font size by default.
+
+- :guilabel:`Field`: dynamically add a field's value.
+
+.. _studio/pdf-reports/elements/table:
+
+Table
+-----
+
+Table elements are used together to create a data table.
+
+- :guilabel:`Data table`: create a table and dynamically add a first column displaying the *Name*
+  values of a :ref:`Many2Many <studio/fields/relational-fields/many2many>` or :ref:`One2Many
+  <studio/fields/relational-fields/one2many>` field on your model.
+
+  .. image:: pdf_reports/data-table.png
+     :align: center
+     :alt: Example of a Data table
+
+- :guilabel:`Field Column`: add a new column to the table displaying the values of a :ref:`Related
+  Field <studio/fields/relational-fields/related-field>` to the one used to create the
+  :guilabel:`Data table`.
+
+- :guilabel:`Text in Cell`: add any text within an existing table cell.
+
+- :guilabel:`Field in Cell`: add, within an existing table cell, the values of a :ref:`Related
+  Field <studio/fields/relational-fields/related-field>` to the one used to create the
+  :guilabel:`Data table`.
+
+- :guilabel:`Subtotal & Total`: add an existing :guilabel:`Total` field's value. If a
+  :guilabel:`Taxes` field exists, the untaxed and taxes amounts are added before the total amount.
+
+.. _studio/pdf-reports/elements/column:
+
+Column
+------
+
+Columns are used to add multiple text :ref:`blocks <studio/pdf-reports/elements/block>` on the same
```

Comment:
important: not necessarily text blocks: you can drop most block elements in there as well. They are "generic containers", "layout guides" (or something like that)